docs/upgrade/UPGRADE-v8.1.0-dev.md

```diff
@@ -79,4 +85,20 @@ There you can find links to upgrade notes for other versions too.
         +       arguments:
         +           $useInlineEditation: false
         ```
+- add support to display date-time values in different timezone ([#1343](https://github.com/shopsys/shopsys/pull/1343))
+    - you can read more about how to [work with display timezone in documentation](/docs/introduction/working-with-date-time-values.md)
+    - you can delete test `tests/ShopBundle/Functional/Twig/DateTimeFormatterExtensionTest.php` as it was moved to the FrameworkBundle, if you don't test any your specific use-case
+    - class `CustomDateTimeFormatterFactory` is deprecated and should not be used anymore
+        - if you have extended this factory to alter configuration of `DateTimeFormatPatternRepository`, extend new `DateTimeFormatPatternRepositoryFactory` instead (as `DateTimeFormatPatternRepository` configuration was moved to this class from the factory)
+        - custom `DateTimeFormatter` have to be created with the DIC directly if needed
+            ```yaml
+            Shopsys\FrameworkBundle\Component\Localization\DateTimeFormatterInterface:
+                class: 'Shopsys\ShopBundle\Component\Localization\DateTimeFormatter'
+            ```
+    - class `DateTimeFormatter` no longer supports extending via `shopsys.entity_extension.map` as it's not entity
```

Comment:
~it's~ it is

Reply:
fixed in ad2dcfae1808de200ae563f458e1aa9fc60420a2